doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -142,7 +142,10 @@ nRF9160: Asset Tracker v2
 nRF9160: Serial LTE modem
 -------------------------
 
-|no_changes_yet_note|
+* Removed:
+
+  * DFU AT commands #XDFUGET, #XDFUSIZE and #XDFURUN are removed.
+    Commands are not usable without a custom application in the target (nRF52 series) device.
```

Comment:
```suggestion
  * DFU AT commands ``#XDFUGET``, ``#XDFUSIZE`` and ``#XDFURUN`` because they were not usable without a custom application in the target (nRF52 series) device.
```
I think combining the sentences makes the most sense to make sure it reads as a reason, then.